packages/muelu/src/MueCentral/MueLu_FactoryManager_def.hpp

```diff
@@ -259,6 +259,8 @@ namespace MueLu {
   void FactoryManager<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Print() const {
     std::map<std::string, RCP<const FactoryBase> >::const_iterator it;
     Teuchos::FancyOStream& fancy = GetOStream(Debug);
+    //auto & fancy = std::cout;// For debugging
```

Comment:
?

Reply:
That was a "I can't figure out how to get the Debug stream to trigger, so I hardcoded cout in instead."

Wanted to leave that in so I remember how to do it again later :)